src/components/Modals/NewCourse/SelectedRequirementEditor.vue

```diff
@@ -1,75 +1,52 @@
 <template>
   <div>
-    <div v-if="chosenRequirementText.length > 0">
+    <div v-if="requirementsThatAllowDoubleCounting.length > 0">
       <div class="newCourse-title">
-        You will let this class fulfill the following requirements(s):
+        This class automatically fulfills the following requirement(s):
       </div>
-      <div class="newCourse-requirements-container">
-        <div class="newCourse-requirements">
-          {{ chosenRequirementText }}
-        </div>
-      </div>
-    </div>
-    <div v-if="requirementsThatAllowDoubleCounting.length > 0">
-      <div class="newCourse-title">This class can auto-fulfill the following requirement(s):</div>
       <div class="newCourse-requirements-container">
         <div class="newCourse-requirements">
           {{ requirementsThatAllowDoubleCounting.join(', ') }}
         </div>
       </div>
     </div>
-    <div v-if="relatedRequirements.length > 0">
+    <div v-else class="newCourse-requirements-container newCourse-requirements">
+      This class does not automatically fulfill any requirements.
+    </div>
+    <div v-if="nonAutoRequirements.length > 0">
       <div class="newCourse-title">This class can fulfill the following requirement(s):</div>
-      <div v-if="!editMode" class="newCourse-requirements-container">
-        <div class="newCourse-requirements">
-          {{ relatedRequirements.map(it => it.name).join(', ') }}
+      <div v-if="!editMode">
+        <div class="newCourse-requirements-container">
+          <!-- eslint-disable-next-line vue/no-v-html  -->
+          <div class="newCourse-title" v-html="nonAutoRequirementsText"></div>
```

Comment:
Can you use a different approach instead of a `v-html` directive? As addressed in #298, v-html directives could possibly lead to a [security issue](https://vuejs.org/v2/guide/security.html#Injecting-HTML) with HTML injection

Reply:
I actually can't think of any other way to do this unfortunately :/ . Since, I'm logically rendering the emboldened text based on data.

Reply:
@hahnbeelee Just pushed my commit for the implementation of an alternative solution. Let me know if you're ok with this. If you are, then this PR should be g2g!